catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsFilterDelegate.java

```diff
@@ -1172,9 +1172,7 @@ private TimeInstantType createTimeInstantType(String type, String date) {
   }
 
   private boolean isValidInputParameters(String propertyName, Object literal) {
-    if (literal == null
-        || StringUtils.isEmpty(propertyName)
-        || StringUtils.isEmpty(literal.toString())) {
+    if (literal == null || StringUtils.isEmpty(propertyName)) {
```

Comment:
this was a small improvement I wanted to slip into this PR.  Empty string values shouldn't be considered invalid in a query.